### DISCUSSION TITLE
Does not work properly with coc.nvim ?

### DISCUSSION THREAD

**Discussion by HeeneFZ** [rmagatti/goto-preview#111] [Q&A] [answered]
Hi! your plugin is pretty good and it real help me a lot. However, it seems to have problem when I try to run it with coc.nvim, there is no reaction. My neovim version is 0.9 and I have ruled out the cause of the shortcut key conflict, what causes this problem and how can it be improved?

**Comment by rmagatti** [comment, marked as answer]
It calls Neovim's built-in LSP functions. So no, it doesn't work with coc.nvim. There's no improvement to be made, it's just not a supported use-case. You could switch to the Neovim LSP and it should then work for you
